Dockerfile

```diff
@@ -17,5 +17,5 @@ RUN python manage.py compilescss
 RUN python manage.py  collectstatic --ignore=*.scss --no-input
 RUN chmod +x ./deploy_files/docker-entrypoint.sh
 
-ENTRYPOINT ["./deploy_files/docker-entrypoint.sh"]
+# ENTRYPOINT ["./deploy_files/docker-entrypoint.sh"]
```

Comment:
This needs to be uncommented, or it won't work when deployed.

Reply:
I'll try to update the project to have separate Dockerfiles for local and deployed so this is no longer an problem.